### ISSUE TITLE
Relative URLs in href attributes are being rewritten incorrectly by wombat.js

### ISSUE THREAD

**Issue by JCCR** [hypothesis/via#117]
I'm trying to get Readium to work in 'via'.
It's not working because the EPUB loading parts of Readium work with a live DOM representation of the EPUB package document.
In the package document there's a file manifest implemented as elements similar to an anchor tag.
`<a href='filename.html'>`
Thankfully pywb doesn't rewrite these relative hrefs, but wombat does..
When calling `anchorElement.getAttribute('href')` 
I'm expecting to get the exact value of the href attribute - `filename.html` - but instead wombat's monkey patching of getAttribute('href') returns `http://filename.html`
Here's a reduced test case:

```
<!DOCTYPE html>
<html>
<head>
</head>
<body>
  <a href='yellow.html'>yellow</a>
  <iframe srcdoc="<a href='banana.html'>banana</a>"></iframe>
  <script>
  alert(document.querySelector('a').getAttribute('href'))

  var iframe = document.querySelector('iframe');
  iframe.addEventListener('load', function() {
    alert(iframe.contentDocument.querySelector('a').getAttribute('href'));
  });
  </script>
</body>
</html>
```
Live version: https://output.jsbin.com/lazulocihu

Expected output:
alert box 1 - `yellow.html`
alert box 2 -`banana.html`

When proxied in via:
https://via.hypothes.is/https://output.jsbin.com/lazulocihu
`https://output.jsbin.com/yellow.html` (pywb rewrites the href attribute here, but for Readium's use case this is fine)
`http://banana.html` (this is the actual issue)

For reference here's how you can load a Readium instance in via:
https://via.hypothes.is/https://readium.firebaseapp.com/?epub=epub_content%2Faccessible_epub_3

**Comment by JCCR** [comment]
I took a quick look at 
https://github.com/ikreymer/pywb/blob/develop/pywb/static/wombat.js

It seems this issue might already be fixed there.

**Comment by ikreymer** [comment]
Yeah, I think Via is still on an older version of pywb.. There's been a lot of fixes since, and it mostly works, though looks like there's some iframe related complexity:

https://webrecorder.io/record/https://readium.firebaseapp.com/?epub=epub_content%2Faccessible_epub_3

Looks like the iframe is being loaded from a `blob:` url..

**Comment by ajpeddakotla** [comment]
Based on feedback from the H team, there's a question on if Via is the best solution to this implementation @JCCR. 

Currently, our via implementation is out of date (as mentioned by @ikreymer), and Via's client-side rewriting is onerous at best. So implementing this in Via wouldn't be an ideal path forward for H. 

Given that context, we may want to consider other ways to resolve the problem. cc @dwhly
